packages/blade/src/components/Checkbox/CheckboxGroup.stories.tsx

```diff
@@ -24,7 +24,7 @@ const Page = (): React.ReactElement => {
       }}
     >
       <Title>Usage</Title>
-      <Sandbox showConsole editorHeight={400} editorWidthPercentage={60}>
```

Comment:
There isn't any way to set width of editor vs preview now. Anyway things look good enough so should be fine.